packages/components/pages/f-offers/src/components/NoOffersFound.vue

```diff
@@ -88,17 +93,35 @@ export default {
     & .c-noOffersFound-searchBox {
         padding-top: spacing(d);
 
+        & input {
+            background-color: $color-white;
+            height: 61px;
+        }
+
         @include media('<=mid') {
             width: 300px;
         }
 
         @include media('<=narrowMid') {
             width: 100%;
+            & input {
+                height: 58px;
+            }
         }
+    }
 
-        & input {
-            background-color: $color-white;
-        }
+    .c-noOffersFound-searchBox--wrapper {
+        position: relative;
+    }
+
+    .c-noOffersFound-searchBox--background {
+        position: absolute;
+        top: spacing(d);
+        left:0;
+        height: 64px;
```

Comment:
Approved but just a question around the set specific height. Has it been tested in all screen sizes?

Reply:
good point have made adjustments